package/base-files/files/bin/ipcalc.sh

```diff
@@ -1,100 +1,93 @@
-#!/usr/bin/awk -f
-
-function bitcount(c) {
-	c=and(rshift(c, 1),0x55555555)+and(c,0x55555555)
-	c=and(rshift(c, 2),0x33333333)+and(c,0x33333333)
-	c=and(rshift(c, 4),0x0f0f0f0f)+and(c,0x0f0f0f0f)
-	c=and(rshift(c, 8),0x00ff00ff)+and(c,0x00ff00ff)
-	c=and(rshift(c,16),0x0000ffff)+and(c,0x0000ffff)
-	return c
-}
+#!/bin/sh
```

Comment:
Always adding quotes is a good idea because checking if there are quotes is a lot easier (and quicker) to do than checking if the code can do without.
I've seen other code in Openwrt being made shellcheck-conformant - assumedly, this should be as well.

Reply:
Done.